### ISSUE TITLE
代码运行问题

### ISSUE THREAD

**Issue by Xiyou521** [zowiezhang/STAS#1]
尊敬的作者：
      你好，我在运行你的代码时，有一个未知的函数名报错，我在所有文件中都没找到MultiAgentAttention函数，请问应该如何解决这个问题？问题如下：
class ShapelyAttention(nn.Module):
    def __init__(self, emb_dim, n_heads, n_agents, sample_num, device, dropout=0.0):
        super().__init__()
        self.emb_dim = emb_dim
        self.device = device
        self.n_agents = n_agents
        self.sample_num = sample_num
        self.phi = MultiAgentAttention(emb_dim, n_heads, n_agents, dropout, device)

**Comment by csr16** [comment]
您好，非常抱歉这么迟回复。目前我们已经修复了这个bug，添加了正确的MultiAgentAttention函数。
